src/status_im2/contexts/quo_preview/list_items/saved_address.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im2.contexts.quo-preview.list-items.saved-address
+  (:require [quo2.core :as quo]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [(preview/customization-color-option {:key :account-color})
+   {:key :blur? :type :boolean}
+   (preview/customization-color-option)])
+
+(defn view
+  []
+  (let [state (reagent/atom {:type                :default
+                             :customization-color :blue
+                             :account-color       :pink
+                             :title               "Alisher Yakupov"
+                             :address             "0x21a...49e"
+                             :on-options-press    #(js/alert "Options button pressed!")})]
+    (fn [] [preview/preview-container
+            {:state                 state
+             :descriptor            descriptor
+             :blur?                 (:blur? @state)
+             :show-blur-background? true
+             :blur-dark-only?       true}
+            [quo/saved-address
+             (merge @state
```

Comment:
`assoc` is more precise here since you always overwrite `:user-props`.

Reply:
Yup, updated with `assoc` instead of `merge`